packages/web-service/src/pages/applications/applications.js

```diff
@@ -1,18 +1,27 @@
 import pageRoute from '../../routes/page-route.js'
 import { APPLICATIONS } from '../../uris.js'
+import { APIRequests } from '../../services/api-requests.js'
 
-const getData = async _request => []
-const setData = async _request => {}
+export const getData = async request => {
+  const journeyData = await request.cache().getData()
+  const { userId } = journeyData
+  const applications = await APIRequests.APPLICATION.findByUser(userId)
+  console.log(JSON.stringify(applications))
+  Intl.DateTimeFormat('en-GB', { year: 'numeric', month: 'long', day: 'numeric' }).format(new Date())
+  return {
+    applications: applications
+      .map(a => ({
+        ...a,
+        lastSaved: Intl.DateTimeFormat('en-GB', { year: 'numeric', month: 'long', day: 'numeric' })
+          .format(new Date(a.updatedAt))
```

Comment:
Is there a reason for using this format? The data standard is to use ISO 8601 for formatting dates

Reply:
It's the display format (e.g. 28 March 2022) specified by Dave on the prototype. I could make this a filter function. Internally I will use Date Objects. I think the Intl functions are the recommended way of formatting Javascript dates now that  Moment is being obsoleted.

Reply:
I think that's my mistake, I hadn't properly processed the fact the object was just for display purposes.